Tests/NIOTests/CodecTest.swift

```diff
@@ -392,8 +392,6 @@ public class ByteToMessageDecoderTest: XCTestCase {
 
         channel.pipeline.context(handlerType: ByteToMessageHandler<PairOfBytesDecoder>.self).then { ctx in
             return channel.pipeline.remove(ctx: ctx)
-        }.map {
-            XCTAssertTrue($0)
         }.whenFailure { error in
```

Comment:
I removed the map to `Bool` here since: first off, it won't happen ever, and 2nd off, if the handler is not removed, it'll throw an error, so the `XCTFail` will trigger